src/test/java/com/google/api/generator/gapic/composer/resourcename/ResourceNameHelperClassComposerTest.java

```diff
@@ -12,12 +12,13 @@
 // See the License for the specific language governing permissions and
```

Comment:
Ditto, these tests exist only in the `composer` parent directory, not in the `resourcenames` subfolder at `master`.

Reply:
see #630